src/gui/tray/UserModel.h

```diff
@@ -69,8 +69,16 @@ public slots:
     void slotRefresh();
     void slotRefreshImmediately();
     void setNotificationRefreshInterval(std::chrono::milliseconds interval);
+    void slotPushNotificationsReady();
+    void slotDisconnectPushNotifications();
```

Comment:
Noticed that just now... do they all need to be public slots? I suspect that whole block could move to private (don't even need the slots marker in that context) and it'd all keep working fine.